json/targeting.yaml

```diff
@@ -389,7 +389,7 @@ type: object
     - description: Bucketing value used in pseudorandom assignment; should be unique
         and stable for each subject of flag evaluation. Defaults to a concatenation
         of the flagKey and targetingKey.
-      $ref: "#/$defs/varRule"
+      $ref: "#/$defs/anyRule"
```

Comment:
Nice, that's easy enough 👍

Reply:
Was skeptical at first since in theory some rules might not return strings, but I don't see a clever way to be so specific in the schema without greatly limiting flexibility

Reply:
@toddbaert any concerns? I believe `anyRule` should cover all valid operations.

Reply:
> Was skeptical at first since in theory some rules might not return strings, but I don't see a clever way to be so specific in the schema without greatly limiting flexibility

We should be casting the value to a string before using it in the hashing function.